src/components/player/index.vue

```diff
@@ -569,9 +576,20 @@ export default {
 			));
 			menu.append(new MenuItem(
 				{
-					label: 'Settings', click() {
-						ipcRenderer.send('settingsModal');
-					}
+					label: (this.websocket && this.websocket.song && this.websocket.song.favorite) ? 'Unfavorite song' : 'Favorite song',
+					click: () => this.toggleFavorite(),
+					enabled: this.loggedIn ? true : false
```

Comment:
You can just do `this.loggedIn` since it returns a boolean value

Reply:
Whoops, true. Unnecessary ternary.